azkaban-db/build.gradle

```diff
@@ -16,6 +16,19 @@
  */
 apply plugin: 'distribution'
 
+task testJar(type: Jar, dependsOn: testClasses) {
+    from sourceSets.test.output
+    classifier = 'test'
+}
+
+configurations {
```

Comment:
@kunkun-tang 
Can you test what @HappyRay suggested. In that case, we may not even have to add a `testJar` task.